### DISCUSSION TITLE
Translation of kinds label

### DISCUSSION THREAD

**Discussion by valamercery** [geovistory/.github#2] [Ideas]
Hi, if I've understood correctly, the kind entities are only associated with the configuration project. In that case, it would be useful to translate them into a few other languages to make them easier to use in non-English-speaking projects. What do you think?
@geovistory/data-team

**Comment by joschne** [comment]
Yes absolutely. IMO translations make sense. But translations will not yet propagate to the UI: Toolbox-streams, responsible for creating the rdfs:label of entities, does create one rdfs:label per entity with the favorite name of the entity. 

In this sense we have to specify how to deal with entity labels and languages.

(Remark: Kinds are managed in the toolbox project "Controlled Vocabularies", not in "configuration").
